fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -954,6 +956,11 @@ private void initAggregateBuiltins() {
         // Ntile
         addBuiltin(AggregateFunction.createAnalyticBuiltin(NTILE,
                 Lists.newArrayList(Type.BIGINT), Type.BIGINT, Type.BIGINT));
+        // Allocate session
+        addBuiltin(AggregateFunction.createAnalyticBuiltin(ALLOCATE_SESSION,
+                Lists.newArrayList(Type.BIGINT, Type.INT), Type.BIGINT, Type.BIGINT));
+        addBuiltin(AggregateFunction.createAnalyticBuiltin(ALLOCATE_SESSION,
```

Comment:
The first argument can be date or datetime, should we support 
allocate_session(timestamp, 5, "minute")? it would be more convenient!  we can handle this variant in AstBuilder.

Reply:
There doesn't seem to be a need for timestamps at the moment, so we can wait until we really need timestamps before we support them